packages/playground/src/calculator/resource_pricing.vue

```diff
@@ -237,7 +242,6 @@ watch([CRU, MRU, SRU, HRU, balance, isCertified, ipv4, currentbalance], async ()
         sru: SRU.value,
         ipv4u: ipv4.value,
         certified: isCertified.value,
-        balance: balance.value,
```

Comment:
now if I try to enter any balance check how much discount will get for this balance, it won't work

IMO, we should disable the balance text field if the switch is enabled without changing the current balance.

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/41957921/fe4f4921-3a25-4c23-8b5b-fb216ddcce6f)

Reply:
- Fixed 

https://github.com/threefoldtech/tfgrid-sdk-ts/assets/54944491/90fadfa9-a48b-43e1-bfbe-ef02909cab01